<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDuplicatesDlgClass</class>
 <widget class="QDialog" name="FindDuplicatesDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FindDuplicatesDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edtRoot"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolBrowse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeFolder">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="gtl::qt::QColorButton" name="btnFindDuplicates">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Find Duplicates</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkRecursive">
           <property name="text">
            <string>Recursive</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtStatus">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="gtl::qt::QColorButton" name="btnDeleteDuplicates">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Delete Duplicates</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="treeFiles"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>gtl::qt::QColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">gtl/qt/QColorButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtRoot</tabstop>
  <tabstop>toolBrowse</tabstop>
  <tabstop>treeFolder</tabstop>
  <tabstop>treeFiles</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
